torch/csrc/jit/tensorexpr/ir_printer.cpp

```diff
@@ -428,43 +419,37 @@ void IRPrinter::visit(const Allocate* v) {
     }
     os() << *dims[i];
   }
-  os() << "]" << std::endl;
+  os() << "]";
 }
 
 void IRPrinter::visit(const Free* v) {
-  emitIndent();
-  os() << "Free(" << *v->buffer_var() << ");" << std::endl;
+  os() << "Free(" << *v->buffer_var() << ");";
 }
 
 void IRPrinter::visit(const Let* v) {
-  emitIndent();
   os() << v->dtype().ToCppString() << " " << *v->var();
   os() << " = " << *v->value();
-  os() << "; " << std::endl;
+  os() << ";";
 }
 
 void IRPrinter::visit(const Cond* v) {
   const Expr* cond = v->condition();
   Stmt* true_stmt = v->true_stmt();
   Stmt* false_stmt = v->false_stmt();
   if (!true_stmt) {
-    emitIndent();
     os() << "if (!" << *cond << ") ";
-    os() << *false_stmt << std::endl;
+    os() << *false_stmt;
```

Comment:
I guess this would be a little yucky if we have a `Cond` with branches that aren't`Block`s.  That doesn't happen, empirically, but...